From bc293dd89fd91045b95b1f88657587c6396eb0bc Mon Sep 17 00:00:00 2001
From: pratik Bhalgat <pratik_bit@codeaurora.org>
Date: Mon, 7 Jul 2014 19:44:33 +0530
Subject: [PATCH 4/4] wlan: Do not invoke wcnss_device_is_shutdown

wcnss_device_is_shutdown depends on the corresponding changes
in the platform driver which are missing, hence do not consider
invoking this function

Change-Id: I7b92a45a9fe3ee4889eb4d634ebfc9ff92c8fd54
---
 CORE/WDI/WPAL/src/wlan_qct_pal_device.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/CORE/WDI/WPAL/src/wlan_qct_pal_device.c b/CORE/WDI/WPAL/src/wlan_qct_pal_device.c
index 3c9518c..3f15b27 100644
--- a/CORE/WDI/WPAL/src/wlan_qct_pal_device.c
+++ b/CORE/WDI/WPAL/src/wlan_qct_pal_device.c
@@ -464,7 +464,7 @@ wpt_status wpalWriteRegister
 {
    /* if SSR is in progress, and WCNSS is not out of reset (re-init
     * not invoked), then do not access WCNSS registers */
-   if (NULL == gpEnv || wcnss_device_is_shutdown() ||
+   if (NULL == gpEnv ||
         (vos_is_logp_in_progress(VOS_MODULE_ID_WDI, NULL) &&
             !vos_is_reinit_in_progress(VOS_MODULE_ID_WDI, NULL))) {
       WPAL_TRACE(eWLAN_MODULE_DAL_DATA, eWLAN_PAL_TRACE_LEVEL_ERROR,
@@ -512,7 +512,7 @@ wpt_status wpalReadRegister
 {
    /* if SSR is in progress, and WCNSS is not out of reset (re-init
     * not invoked), then do not access WCNSS registers */
-   if (NULL == gpEnv || wcnss_device_is_shutdown() ||
+   if (NULL == gpEnv ||
         (vos_is_logp_in_progress(VOS_MODULE_ID_WDI, NULL) &&
             !vos_is_reinit_in_progress(VOS_MODULE_ID_WDI, NULL))) {
       WPAL_TRACE(eWLAN_MODULE_DAL_DATA, eWLAN_PAL_TRACE_LEVEL_ERROR,
@@ -563,7 +563,7 @@ wpt_status wpalWriteDeviceMemory
 {
    /* if SSR is in progress, and WCNSS is not out of reset (re-init
     * not invoked), then do not access WCNSS registers */
-   if (NULL == gpEnv || wcnss_device_is_shutdown() ||
+   if (NULL == gpEnv ||
         (vos_is_logp_in_progress(VOS_MODULE_ID_WDI, NULL) &&
             !vos_is_reinit_in_progress(VOS_MODULE_ID_WDI, NULL))) {
       WPAL_TRACE(eWLAN_MODULE_DAL_DATA, eWLAN_PAL_TRACE_LEVEL_ERROR,
@@ -607,7 +607,7 @@ wpt_status wpalReadDeviceMemory
 {
    /* if SSR is in progress, and WCNSS is not out of reset (re-init
     * not invoked), then do not access WCNSS registers */
-   if (NULL == gpEnv || wcnss_device_is_shutdown() ||
+   if (NULL == gpEnv ||
         (vos_is_logp_in_progress(VOS_MODULE_ID_WDI, NULL) &&
             !vos_is_reinit_in_progress(VOS_MODULE_ID_WDI, NULL))) {
       WPAL_TRACE(eWLAN_MODULE_DAL_DATA, eWLAN_PAL_TRACE_LEVEL_ERROR,
-- 
1.8.2.1

